### ISSUE TITLE
Block User Agent

### ISSUE THREAD

**Issue by jiujou00** [Manvel/Privacy-Manager#39]
When I check the option block user agent and test it I get "Browser headers different. User-Agent that your browser sends differs from that we checked via Javascript. You are trying to hide your actual browser."

**Comment by Manvel** [comment]
Right, in Privacy Manager User-Agent blocking is working on the network level.
So when you visit a website your browser send information within the request which contain User Agent.
The problem is, that JavaScript which runs on your machine has access to that information as well, so the test you are doing probably load the JavaScript and run on your machine. There is no mechanism in Privacy Manager yet to prevent that. I'll keep this issue open to communicate this better in future or find ways to block User Agent on the client machine as well, though I can think of a good solution for handling that right now.
